README.md

```diff
@@ -94,7 +94,8 @@ accelerate config
 You can then evaluate a model using data parallelism as follows:
 
 ```shell
-accelerate launch --multi_gpu --num_processes=<num_gpus> run_evals_accelerate.py \
+accelerate launch --multi_gpu --num_processes=<num_gpus> -m \
```

Comment:
I'm not fond of `accelerate ... lighteval accelerate`

Reply:
I agree but not sure how we could really differenciate between nanotron and accelerate launchers

Reply:
wdym ?

Reply:
I don't think we'll ever get `accelerate launch ... lighteval nanotron`

Reply:
So we could assume that if `accelerate launch` is used, then it's lighteval accelerate?